tasks/main.yml

```
@@ -97,6 +70,8 @@
     priv: "{{ database_name }}.*:ALL,GRANT"
     login_user: "{{ database_root_user }}"
     login_password: "{{ database_root_password }}"
```

Password may be unset. This would handle this case.

```suggestion
    login_password: "{{ database_root_password | default(omit) }}"
```